Error when selecting multiple audio files

wx._core.wxAssertionError: C++ assertion ""!HasFlag(wxFD_MULTIPLE)"" failed at ext\wxWidgets\include\wx/filedlg.h(109) in wxFileDialogBase::GetPath(): When using wxFD_MULTIPLE, must call GetPaths() instead

--- a/photofilmstrip/core/GPlayer.py
+++ b/photofilmstrip/core/GPlayer.py
@@ -115,7 +115,8 @@ class GPlayer:
         self.Close()
 
     def Close(self):
-        self.__pipeline.send_event(Gst.Event.new_eos())
+        if self.__pipeline:
+            self.__pipeline.send_event(Gst.Event.new_eos())
 
     def GetLength(self):
         return self.__length

--- a/photofilmstrip/gui/DlgConfigureAudio.py
+++ b/photofilmstrip/gui/DlgConfigureAudio.py
@@ -15,7 +15,6 @@ from photofilmstrip.core.AudioPlayer import AudioPlayer
 
 from photofilmstrip.gui.ctrls.PnlDlgHeader import PnlDlgHeader
 from photofilmstrip.lib.util import FILE_EXTENSIONS_AUDIO, GetDataDir
-from docutils.parsers.rst.directives import path
 
 
 class DlgConfigureAudio(wx.Dialog):
@@ -187,10 +186,8 @@ class DlgConfigureAudio(wx.Dialog):
                             _("Audio files") + " (*.*)|*.*",
                             wx.FD_OPEN | wx.FD_MULTIPLE)
         if dlg.ShowModal() == wx.ID_OK:
-            path = dlg.GetPath()
-            Settings().SetAudioPath(os.path.dirname(path))
-
             for path in dlg.GetPaths():
+                Settings().SetAudioPath(os.path.dirname(path))
                 self.__AddAudioItem(path, select=True)
 
         dlg.Destroy()

